js/get_help_now.js

```diff
@@ -1,26 +1,42 @@
+/**
+ * showGetHelpActionSection() displays the page section
+ * where the user has options to seek help
+ * such as call or message
+ */
+function showGetHelpActionSection () {
+	var getHelpSection = $('#get-help-section'),
+		getHelpActionSection = $('#get-help-action-section');
+    getHelpSection.fadeOut('slow');
+    getHelpActionSection.delay(500).fadeIn('slow');
+}
+
+/**
+ * hideGetHelpActionSection() hides the get help page section
+ * and navigate to other section
+ */
+function hideGetHelpActionSection() {
+	var getHelpSection = $('#get-help-section'),
+		getHelpActionSection = $('#get-help-action-section');
+    getHelpActionSection.fadeOut('slow');
+    getHelpSection.delay(500).fadeIn('slow');
+    setTimeout(function() {
+		$('#content-container').addClass('hide');
+    	$('#content-container > div > div').addClass('hide');
+    }, 600);
+}
 $(document).ready(function() {
 	$('.contact-button').on('click', function() {
-		var thisId = $(this).attr('id'),
-			getHelpSection = $('#get-help-section'),
-			getHelpActionSection = $('#get-help-action-section');
+		var thisId = $(this).attr('id');
```

Comment:
This is a widely used term in JS and Jquery. Just following the coding conventions and naming style, specifying that we are referencing the ID of 'this' particular element.

Reference: [What is this?](http://www.learningjquery.com/2007/08/what-is-this)